src/physics/manufactured_solution.h

```diff
@@ -503,11 +503,38 @@ class ManufacturedSolutionQuadratic
     std::array<real, dim> alpha_diag; ///< Diagonal hessian component scaling
 };
 
+
+/// Sum of example functions manufactured solution (was used in Manufactured Solution presentation).
```

Comment:
I see, but still, a more descriptive name would be nice

Reply:
fixed as per issue https://github.com/dougshidong/PHiLiP/issues/126